app/lib/payment/braintree_blue_errors_handler.rb

```diff
@@ -23,17 +25,17 @@ def messages
     private
 
     def collect_messages
-      notify_no_errors_given!
-      messages = @result.errors.map(&:message)
-      messages << CREDIT_CARD_VERIFICATION_MESSAGE if @result.credit_card_verification
+      notify_no_errors_given
+      messages = result.errors.map(&:message)
+      messages << CREDIT_CARD_VERIFICATION_MESSAGE if result.credit_card_verification
       messages.compact!
       messages
     end
 
     # This will notify that some errors are not sent by Braintree
-    def notify_no_errors_given!
-      return if @result.errors.any?
-      error = BraintreeResultError.new @result
+    def notify_no_errors_given
+      return if result.errors.any?
```

Comment:
Is there an attr_reader for result defined? :scream:

Reply:
Yup, you are right, there is :smile: 
![image](https://user-images.githubusercontent.com/11318903/70253462-910e2780-1783-11ea-9741-1933967330c1.png)
![image](https://user-images.githubusercontent.com/11318903/70253492-9bc8bc80-1783-11ea-873b-98c27daeb7db.png)